### PR TITLE
Made Users Page and add Link

### PR DESCRIPTION
**Pull request by kenneth-blanton**
component in navbar.tsx

### CODE REVIEW

File: wombat-admin/src/app/users/page.tsx
Comment (joshuaow91) [inline wombat-admin/src/app/users/page.tsx:1]: You can map through the users section like this

```
const users = [
    { id: 1, name: 'Lindsay Walton', role: 'Front End Developer', dateCreated: '2021-08-25' },
    { id: 2, name: 'Stephanie Curry', role: 'Back End Developer', dateCreated: '2021-10-17' },
    { id: 3, name: 'John Doe', role: 'UI/UX Designer', dateCreated: '2021-11-11' }
];
```

```


{users.map((user) => (
  <tr key={user.id}>
   <td className="whitespace-nowrap py-5 pl-4 pr-3 text-sm sm:pl-0">
    {user.id}
   </td>
   <td className="whitespace-nowrap px-3 py-5 text-sm text-gray-500">
   <div className="font-medium text-gray-900">
   {user.name} 
   </div>
</td>
<td className="whitespace-nowrap px-3 py-5 text-sm text-gray-500">
{user.role}
</td>
<td className="whitespace-nowrap px-3 py-5 text-sm text-gray-500">
{user.dateCreated}
</td>
<td className="whitespace-nowrap px-3 py-5 text-sm text-gray-500">
<button
type="button"
className="text-white hover:bg-teal-500 bg-teal-600 px-2 py-1 text-xs font-medium rounded-md"
>
View
</button>
</td>
</tr>
))}

```


that will set us up to replace the hardcodes object array with an api call when the endpoint is created. also not sure if a div should be a child of a TD

Comment (allaboutmike) [reply wombat-admin/src/app/users/page.tsx:1]: Agree that we should use a map here, as that has been the approach for the other pages. Let's switch to that.

Nothing wrong with putting a div, or other elements, within a table cell. Span would also work to be honest if we need a container for the styles.

**Review by allaboutmike — CHANGES_REQUESTED** [review]
Can we change to a map in the users/page.tsx file to make it easier to hook into the backend later?

**Review by joshuaow91 — APPROVED** [review]
Looking good!!!

**Review by allaboutmike — APPROVED** [review]

**Pull request merged**